plugins/AngleMeasure/src/AngleMeasure.cpp

```diff
@@ -17,6 +17,9 @@
  * Foundation, Inc., 51 Franklin Street, Suite 500, Boston, MA  02110-1335, USA.
  */
 
+#define PAN_AND_ZOOM
```

Comment:
Don't just define it. Comment what your intent for this is.

Reply:
this definition switches to the "new" behavior - otherwise the "old" behavior is activated

Reply:
Will the next developer know this two years from now?